lib/brain.js

```diff
@@ -9,6 +9,7 @@ const Rx = require('rxjs/Rx')
 const pify = require('pify')
 const pAny = require('p-any')
 const pSettle = require('p-settle')
+const Bluebird = require('bluebird')
```

Comment:
Let's settle on the promise-fun variants for now: https://github.com/sindresorhus/promise-fun

https://github.com/sindresorhus/delay

Reply:
ok